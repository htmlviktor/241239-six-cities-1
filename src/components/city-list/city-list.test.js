import React from 'react';
import renderer from 'react-test-renderer';
import CityList from './city-list.jsx';

const mock = new Set([
  `Amsterdam`,
  `Budapesht`,
  `Wroclaw`
]);

it(`Test CityList Component`, () => {
  const tree = renderer
    .create(<CityList
      onChangeCity={() => {}}
      cities={mock}
    />).toJSON();

  expect(tree).toMatchSnapshot();
});
