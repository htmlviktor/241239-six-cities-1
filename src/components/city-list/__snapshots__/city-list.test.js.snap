// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test CityList Component 1`] = `
<ul
  className="locations__list tabs__list"
>
  <li
    className="locations__item"
  >
    <a
      className="locations__item-link tabs__item"
      href="#"
      onClick={[Function]}
    >
      <span>
        Amsterdam
      </span>
    </a>
  </li>
  <li
    className="locations__item"
  >
    <a
      className="locations__item-link tabs__item"
      href="#"
      onClick={[Function]}
    >
      <span>
        Budapesht
      </span>
    </a>
  </li>
  <li
    className="locations__item"
  >
    <a
      className="locations__item-link tabs__item"
      href="#"
      onClick={[Function]}
    >
      <span>
        Wroclaw
      </span>
    </a>
  </li>
</ul>
`;
